twitter-langchain/twitter_langchain/twitter_toolkit.py

```diff
@@ -47,7 +55,7 @@ class TwitterToolkit(BaseToolkit):
 
         .. code-block:: none
 
-            # TODO: Add list of available tools.
+            twitter_post_text
```

Comment:
this needs to match the name of the action.

I think we should name this `post_tweet` and rename all of the methods accordingly.

Reply:
roger that!